### ISSUE TITLE
String value of properties declaration

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#485]
### Issue
Running code below should fail or report validation error because according to RAML 1.0 spec provided RAML input is invalid.

The spec says:
> The properties declaration is a map of keys and values.

Spec link:
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#property-declarations

### AMF Version
4.0.0-SNAPSHOT

### Code
```sh
$ java -jar amf-4.0.0-SNAPSHOT.jar validate -in "RAML 1.0" -mime-in "application/yaml" -p "RAML" file://api.raml
{
  "@type": "http://www.w3.org/ns/shacl#ValidationReport",
  "http://www.w3.org/ns/shacl#conforms": true
}
```

### api.raml
```raml
#%RAML 1.0
title: Example of request bodies
mediaType: application/json
/users:
  post:
    body:
      application/json:
        properties: asd
```

**Comment by BlockLucas** [comment]
https://www.mulesoft.org/jira/browse/APIMF-1520

**Comment by gutee** [comment]
This is fixed and stated for 3.3.0 release
